app/cdash/tests/test_submissionvalidation.php

```diff
@@ -0,0 +1,39 @@
+<?php
+require_once dirname(__FILE__) . '/cdash_test_case.php';
+
+
+
+class SubmissionValidationTestCase extends KWWebTestCase
```

Comment:
It would be better to make this a feature test under `/tests/Feature`.  Getting a submission method in a feature test will require https://github.com/Kitware/CDash/pull/2578 though, which of course requires us to figure out what is causing errors on your system.  What do you think about delaying this until https://github.com/Kitware/CDash/pull/2578 is merged and then converting this to a feature test which uses the new submission method?

Reply:
That seems more than reasonable to me.

Reply:
#2578 has been merged, so you can now use the new submission trait in a feature test instead of this legacy test class.